app/themes/colorPalette.js

```diff
@@ -0,0 +1,148 @@
+export default {
+  primary: {
+    background: '#FFFFFF',
+    purpleLight: '#DCE0F9',
+    purpleMedium: '#617DFF',
+    purpleDark: '#271B46',
+  },
+  secondary: {
+    coral: '#F9706B',
+    blue: '#40EBF9',
+    green: '#4CE791',
+    orange: '#F95F3B',
+    peach: '#FCCE9F',
+  },
+  neutrals: {
+    white: '#FFFFFF',
+    lightestGrey: '#F9F9F9',
+    lightGrey: '#EDEDED',
+    mediumGrey: '#979797',
+    darkGrey: '#606060',
+    black: '#000000',
+  },
+  extended: {
```

Comment:
I know you're just pulling these from the design sheet, but the fact that there aren't an equal number of options in each of these arrays seems like it could get annoying. Thinking along the lines of "I need a darker orange but I can't remember how big the oranges array is to know how far I should go." Not a comment on the code for this review, but a comment about a conversation that might be had with design.

Reply:
Yeah, may be worth discussing at our next frontend/design huddle.